web/satellite/src/components/accessGrants/CreateAccessModal.vue

```diff
@@ -0,0 +1,550 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="create-access">
+        <div class="create-access__modal-container">
+            <form>
+                <div class="create-access__modal-container__header-container">
+                    <h2 class="create-access__modal-container__header-container__title">Create Access</h2>
+                    <div
+                    class="create-access__modal-container__header-container__close-cross-container" @click="onCloseClick">
+                        <CloseCrossIcon />
+                    </div>
+                </div>
+                <div class="create-access__modal-container__body-container">
+                    <TypesIcon class="create-access__modal-container__body-container__type-icon"/>
+                    <div class="create-access__modal-container__body-container__type">
+                        <p>Type</p>
+                        <div>
+                            <input 
+                            v-model="checkedType"
+                            value="access"
+                            type="radio" 
+                            id="acess-grant-check"
+                            name="type" 
+                            :checked="this.checkedType === 'access'"/>
+                            <label for="acess-grant-check">Access Grant</label>
+                        </div>
+                        <div>
+                            <input 
+                            v-model="checkedType"
+                            value="s3"
+                            type="radio" 
+                            id="s3-check"
+                            name="type" 
+                            :checked="this.checkedType === 's3'"/>
+                            <label for="s3-check">S3 Credentials</label>
+                        </div>
+                        <div>
+                            <input
+                            v-model="checkedType"
+                            value="api"
+                            type="radio" 
+                            id="api-check"
+                            name="type" 
+                            :checked="this.checkedType === 'api'"
+                            @click="this.checkedType = 'api'"/>
+                            <label for="api-check">API Access</label>
+                        </div>
+                    </div>
+                    <NameIcon class="create-access__modal-container__body-container__name-icon"/>
+                    <div class="create-access__modal-container__body-container__name">
+                        <p>Name</p>
+                        <input
```

Comment:
we have common components for inputs
+ tabs

Reply:
Tabs were fixed by lint.

Andrew has already begun his work down stream of this step. Because of this I hesitate to change the input because of the impact to his work. This may be something to refactor once he is complete.